app/controllers/api/v1/rooms_controller.rb

```diff
@@ -20,6 +20,11 @@ def show
         render_json data: @room, status: :ok
       end
 
+      def destroy
+        Room.destroy_by(friendly_id: params[:friendly_id])
+        render_json status: :ok
```

Comment:
To further simplify this, `render_json` by default returns `render json: { data: [], errors: [] }, status: :ok, include: nil`